src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -664,4 +676,61 @@ private String postRequest(String path, NameValuePair[] params) throws IOExcepti
         }
         return branches;
     }
+
+    private <T> T parse(String response, Class<T> clazz) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
+        return mapper.readValue(response, clazz);
+    }
+
+    public Iterable<SCMFile> getDirectoryContent(BitbucketSCMFile parent) throws IOException, InterruptedException {
+        StringBuilder url = new StringBuilder(V1_API_BASE_URL);
```

Comment:
Use V2 as recently V1 as removed

Reply:
Seems like it's the same path:

Reply:
Yep, I started to work on this before V2 support was available in the master branch :(